browser/agent-browser/src/agent-browser.ts

```
@@ -42,6 +42,7 @@ import type { CreateAgentBrowserThreadManager } from './thread-manager';
 import { createAgentBrowserTools } from './tools';
 import type { BrowserConfig } from './types';
 import { getBrowserPid } from './utils';
+import { installPlaywrightLinuxDeps } from './playwright-deps';
```

ESLint

[error] `./playwright-deps` import should occur before type import of `./schemas`

(import/order)